docs/getting_started/tutorial10.rst

```diff
@@ -0,0 +1,44 @@
+Your First CodeRed Website: Part 10 -- Password-Protected Pages
+===============================================================
+
+Finally, let's talk about how to **password-protect** pages that you only want those with proper 
+permissions to be able to access when they visit the website. You may want to hide a document page 
+for tenants  or an information page for customers behind a password, and there is an easy way to 
```

Comment:
Tenant example is not relevant here. A good example for cupcake shop might be a private document with bulk pricing info, or something to that effect.

Reply:
fixed